sleap/gui/dialogs/formbuilder.py

```diff
@@ -458,14 +465,16 @@ def add_item(self, item: Dict[Text, Any]):
             self.addRow(field)
             return
 
-        # double: show spinbox (number w/ up/down controls)
-        elif item["type"] == "double":
-            
-            #If the label of the item is Loss Weight then we will set the min and max to 0 and 1000
-            if item["label"] == "Loss Weight":
+        elif (
+            item["type"] == "double"
+        ):  # double: show spinbox (number w/ up/down controls)
+
+            if (
+                item["label"] == "Loss Weight"
+            ):  # Set min/max to 0-1000 if label is "Loss Weight"
```

Comment:
This bit of code is pretty self explanatory...
```suggestion
            if item["label"] == "Loss Weight": 
```
 if you really want to keep the comment make it a block comment. Why? The line you put the comment on isn't really what that line does is it? It's what happens in the following block. (Also, side note, the biggest thing is setting the ExponentialSPinBox, not changing the min value.)